host-interaction/process/inject/inject-shellcode-using-mapping-injection.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: inject shellcode using Mapping Injection
```

Comment:
```suggestion
    name: inject shellcode using a file mapping object
```

imho, "Mapping Injection" isn't a a well known term and the technique seems to be a fairly straightforward application of file mapping objects, so let's just describe it as that

Reply:
Sure, that makes perfect sense. I submitted the renamed rule.